--- a/compiler/src/test/java/com/fluxtion/compiler/builder/factory/GraphOfInstancesTest.java
+++ b/compiler/src/test/java/com/fluxtion/compiler/builder/factory/GraphOfInstancesTest.java
@@ -10,6 +10,7 @@ import org.junit.Test;
 import org.yaml.snakeyaml.TypeDescription;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.Constructor;
+import org.yaml.snakeyaml.LoaderOptions;
 
 import java.util.ArrayList;
 import java.util.List;
@@ -75,7 +76,7 @@ public class GraphOfInstancesTest extends MultipleSepTargetInProcessTest {
         sep(c -> {
-            Constructor constructor = new Constructor();
+            Constructor constructor = new Constructor(new LoaderOptions());
             constructor.addTypeDescription(new TypeDescription(DoubleSum.class, "!doubleSum"));
             constructor.addTypeDescription(new TypeDescription(StringHandler.class, "!stringHandler"));
             Yaml yaml = new Yaml(constructor);
@@ -111,7 +112,7 @@ public class GraphOfInstancesTest extends MultipleSepTargetInProcessTest {
         sep(c -> {
-            Constructor constructor = new Constructor();
+            Constructor constructor = new Constructor(new LoaderOptions());
             constructor.addTypeDescription(new TypeDescription(DoubleSum.class, "!doubleSum"));
             constructor.addTypeDescription(new TypeDescription(StringHandler.class, "!stringHandler"));
             Yaml yaml = new Yaml(constructor);
