diff --git a/src/main/java/io/qameta/allure/maven/AllureCommandline.java b/src/main/java/io/qameta/allure/maven/AllureCommandline.java
index 51fb3b3..ee48716 100644
--- a/src/main/java/io/qameta/allure/maven/AllureCommandline.java
+++ b/src/main/java/io/qameta/allure/maven/AllureCommandline.java
@@ -15,7 +15,7 @@
  */
 package io.qameta.allure.maven;
 
-import net.lingala.zip4j.core.ZipFile;
+import net.lingala.zip4j.ZipFile;
 import net.lingala.zip4j.exception.ZipException;
 import org.apache.commons.exec.CommandLine;
 import org.apache.commons.exec.DefaultExecutor;
